sample.config.toml

```diff
@@ -10,7 +10,7 @@
   default = ''
 
 [remotes]
-  default = ''
+  default = { name = '', url = '' }
```

Comment:
What's about a dedicated section which the generic helper create and provide them automatically?